Main.java

```diff
@@ -3,74 +3,145 @@
 import java.io.*;
 import java.nio.file.Files;
 import java.nio.file.Paths;
+import java.sql.*;
 import java.util.StringTokenizer;
 
+import static com.company.Main.ERROR_MESSAGE;
+import static com.company.Main.statement;
+
 public class Main {
 
-    final static String FILE_TYPE_TXT = ".txt";
-    final static String FILE_TYPE_JSON = ".json";
-    final static String REMOVE_SPECIAL_CHAR = "\\p{Punct}";
-    final static String SINGLE_SPACE = "";
+    static final String FILE_TYPE_TXT = ".txt";
+    static final String FILE_TYPE_JSON = ".json";
+    static final String REMOVE_SPECIAL_CHAR = "[^a-zA-Z0-9 -]";
+    static final String SINGLE_SPACE = "";
+    static String ERROR_MESSAGE;
+    static PreparedStatement statement;
+    static String textFilePath;
+    static String searchKeyword;
+
 
     public static void main(String[] args) {
 
         if (args.length != 2) {
-            System.out.println("File path and Search word cannot be empty");
+            ERROR_MESSAGE = "Provide valid filepath and search keyword";
+            System.out.println(ERROR_MESSAGE);
+            Database.wordSearchFailure(null, null, ERROR_MESSAGE);
             return;
         }
-        String textFilePath = args[0];
-        String searchKeyword = args[1];
+        textFilePath = args[0];

```

Comment:
textFilePath & searchKeyword do not need to be static field, keep them as local variables itself.